jekyll/_cci2/openid-connect-tokens.adoc

```diff
@@ -54,7 +54,11 @@ The OpenID Connect ID token contains the following standard https://openid.net/s
 | The issuer. The issuer is specific to the CircleCI organization in which the job is being run. Its value is `"https://oidc.circleci.com/org/ORGANIZATION_ID"`, a string, where `ORGANIZATION_ID` is a UUID identifying the current job's project's organization.
 
 | `sub`
-| The subject. This identifies who is running the CircleCI job and where. Its value is `"org/ORGANIZATION_ID/project/PROJECT_ID/user/USER_ID"`, a string, where `ORGANIZATION_ID`, `PROJECT_ID`, and `USER_ID` are UUIDs that identify the CircleCI organization, project, and user, respectively. The user is the CircleCI user that caused this job to run.
+a| The subject. This identifies who is running the CircleCI job and where. `$CIRCLE_OIDC_TOKEN_2` also includes information about the source of change.
+
+For `$CIRCLE_OIDC_TOKEN` its value is: `"org/ORGANIZATION_ID/project/PROJECT_ID/user/USER_ID"`, a string, where `ORGANIZATION_ID`, `PROJECT_ID`, and `USER_ID` are UUIDs that identify the CircleCI organization, project, and user, respectively. The user is the CircleCI user that caused this job to run. 
+
+For `$CIRCLE_OIDC_TOKEN_2` its value is: `"org/ORGANIZATION_ID/project/PROJECT_ID/user/USER_ID/vcs-origin/VCS_ORIGIN/vcs-ref/VCS_REF"`, a string, where `ORGANIZATION_ID`, `PROJECT_ID`, and `USER_ID` are UUIDs that identify the CircleCI organization, project, and user, respectively. The user is the CircleCI user that caused this job to run. `VCS_ORIGIN` and `VCS_REF` are strings that identify the repo URL and reference to the change that caused the job to run.
```

Comment:
I tried to clarify what the VCS elements do, this needs checking @nathanfishcircleci